sw/airborne/modules/dcf/dcf.h

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2017  Hector Garcia de Marina
+ *
+ * This file is part of paparazzi.
+ *
+ * paparazzi is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * paparazzi is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with paparazzi; see the file COPYING.  If not, write to
+ * the Free Software Foundation, 59 Temple Place - Suite 330,
+ * Boston, MA 02111-1307, USA.
+ *
+ */
+
+/** @file dcf.h
+ *
+ *  Distributed circular formation algorithm
+ */
+
+#ifndef DCF_H
+#define DCF_H
+
+#include "std.h"
+
+typedef struct {
+  float k;
+  float radius;
+  uint16_t timeout;
+  float theta;
+} dcf_con;
```

Comment:
We don't typedef structs in paparazzi. define the strtuct then reference it with struct dcf_con. This helps to reinforce that these variables are structs.

Reply:
Is this really a strict rule or a suggestion? mmm I was doing the same multiple times with the GVF algorithm (for each trajectory and for the control variables). I mean, I can stop this practice and change it here, but for the GVF it will be quite some work...